apps/central/src/components/admin/ManageSettings.vue

```diff
@@ -0,0 +1,154 @@
+<template>
+  <div>
+    <h3>Manage system settings</h3>
+
+    <table class="table table-hover table-bordered bg-white">
+      <thead>
+        <th style="width: 1px">
+          <IconAction icon="plus" @click="handleCreateRequest" />
+        </th>
+        <th>key</th>
+        <th>value</th>
+      </thead>
+      <tbody v-if="settings">
+        <tr v-for="setting in settings" :key="setting.key">
```

Comment:
Elsewhere in emx2 I decided to not have this kind of generic 'all settings are possible' user interface but instead a specific user interface that only provides UX for the settings that we actually support. Otherwise we would need a lot of documentation and also help to the user to create the right values. Also this then immediately serves as functional documentation.